framework/src/executioners/PicardSolve.C

```diff
@@ -375,6 +380,20 @@ PicardSolve::solve()
   return converged;
 }
 
+bool
+PicardSolve::autoAdvance() const
+{
+  bool auto_advance = !(_has_picard_its && _problem.isTransient());
```

Comment:
Could you add your findings right here? So we know why we have auto on when there is no picard?

Reply:
Yes I'll add some good documentation to the `.md` files of what I've outlined in my comments here on github.

Note that I have this PR on draft, so I'm guessing the auto merge label isn't going to have an effect... Actually this could be an interesting test of CIVET :smile: